README.md

```diff
@@ -2,10 +2,29 @@
 
 An OpenShift based project that provides a local environment for testing OSBS components.
 
-Currently, this environment is not yet complete (except for the Koji part).
 
+## Basic usage
 
-## How-to
+Run a simple container build on your OSBS-Box:
+
+```bash
+$ # Get URL of docker registry
+$ OSBS_REGISTRY=$(oc -n osbs-registry get route osbs-registry --output jsonpath='{ .spec.host }')
+
+$ # Copy base image for container build to docker registry
+$ skopeo copy docker://registry.fedoraproject.org/fedora:30 \
```

Comment:
skopeo doesn't copy manifests-lists, we use skopeo-lite to copy image with manifests list

Reply:
You can address it in separate PR.

Reply:
Addressed

Reply:
FWIW I believe [skopeo v0.1.40](https://github.com/containers/skopeo/releases/tag/v0.1.40) adds a `--all` flag to do just that.